src/routers/datatypeRouters/edges/variants_variants.ts

```diff
@@ -52,9 +60,9 @@ const variantLDQueryFormat = z.object({
 export async function findVariantLDSummary(input: paramsFormatType): Promise<any[]> {
   const originalPage = input.page as number
 
-  let limit = QUERY_LIMIT
+  let limit = 15
   if (input.limit !== undefined) {
-    limit = (input.limit as number <= MAX_PAGE_SIZE) ? input.limit as number : MAX_PAGE_SIZE
+    limit = (input.limit as number <= 50) ? input.limit as number : MAX_PAGE_SIZE
```

Comment:
Need to change 50 back to MAX_PAGE_SIZE and assign value 50 to MAX_PAGE_SIZE?

Reply:
good catch! added the change